SearchQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -55,40 +62,47 @@ Then run the command `docker images`, you will have following images ready:
 2. `opea/web-retriever:latest`
 3. `opea/reranking:latest`
 4. `opea/llm-textgen:latest`
-5. `opea/searchqna:latest`
-6. `opea/searchqna-ui:latest`
+5. `opea/nginx:latest`
+6. `opea/searchqna:latest`
+7. `opea/searchqna-ui:latest`
 
 ## 🚀 Set the environment variables
 
-Before starting the services with `docker compose`, you have to recheck the following environment variables.
+```bash
+cd GenAIExamples/SearchQnA/docker_compose/
+```
+
+Before starting the services with `docker compose`, you have to recheck the following environment variables. Create a file to store
+your custom environment variables.
+
+```bash
+nano setenv_searchqna.sh
+```
+
+And paste in the following near top:
```

Comment:
We directly write these into set_env.sh, with comments

Reply:
This could simplify things.   I just want to double check, when I read [ChatQnA README.md](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA),  `Quick Start: 1.Setup Environment Variable`, the steps 1, 2 are separate from step 3 (`source ./set_env.sh`).   Is the separation intentional?

Reply:
Adjusted README so that it is more explicit to not save keys in plain text .